src/main/java/pl/allegro/tech/discovery/consul/recipes/watch/ConsulWatcher.java

```diff
@@ -91,33 +88,20 @@ private void watchAtIndex(HttpUrl endpoint, ConsulLongPollCallback callback, lon
 
             Request request = new Request.Builder().get().url(url).build();
             httpClient.newCall(request).enqueue(callback);
-        } else {
-            logger.info("Stopping long poll at endpoint {}", endpoint);
```

Comment:
Will we see this log with your change? I'd rather see "starting", "stopping" in my app logs.

Reply:
The log is back in it's place